README.md

```diff
@@ -1,24 +1,89 @@
-# README
 
```

Comment:
- [ ] It would be better if we give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/). We can achieve this in the Acknowledgements section of our README file